src/invoices/InvoiceDetails/InvoiceDetails.js

```diff
@@ -78,9 +79,13 @@ class InvoiceDetails extends Component {
       deleteInvoice,
       onClose,
       invoice,
+      parentResources,
     } = this.props;
     const { sections, showConfirmDelete } = this.state;
     const vendorInvoiceNo = invoice.vendorInvoiceNo;
+    const users = get(parentResources, 'users.records', []);
```

Comment:
does it make sense to load only one user for related invoice?

Reply:
Yes. Fixed